Test.AspNetCore.ServerSentEvents/Middleware/Infrastructure/SubjectUnderTestHelper.cs

```diff
@@ -27,15 +27,16 @@ internal static class SubjectUnderTestHelper
             IServerSentEventsClientIdProvider serverSentEventsClientIdProvider = null,
             IServerSentEventsNoReconnectClientsIdsStore serverSentEventsNoReconnectClientsIdsStore = null,
             ServerSentEventsService serverSentEventsService = null,
-            ServerSentEventsOptions options = null)
+            ServerSentEventsOptions options = null,
+            bool validateConnectionResult = true)
```

Comment:
This shouldn't be here, providing this should be done by providing proper `ServerSentEventsService`.